main.go

```diff
@@ -28,15 +29,17 @@ type RepoSummary struct {
 func main() {
 
 	var (
-		orgName, userName, token, tokenFile string
-		days                                int
-		punchCard, byRepo                   bool
+		orgName, userName, token, tokenFile, repoList, reposFile string
+		days                                                     int
+		punchCard, byRepo                                        bool
 	)
 
 	flag.StringVar(&orgName, "org", "", "Organization name")
 	flag.StringVar(&userName, "user", "", "User name")
 	flag.StringVar(&token, "token", "", "GitHub token")
 	flag.StringVar(&tokenFile, "token-file", "", "Path to the file containing the GitHub token")
+	flag.StringVar(&repoList, "include", "", "List of repos you want stats for eg. 'org/repo1,org/repo2'")
```

Comment:
instead of a list of slugs, could it perhaps be a glob, so that patterns can be used?

Reply:
So to clarify, are you suggesting something along the lines of `-include user/repo-*` to match a bunch of repos as opposed to each of them individually?

Reply:
Yes, exactly.

Reply:
I have hundreds of repos i maintain, and globbing is absolutely necessary to be able to target them - they’re often organized by naming convention (usually a prefix), and without globs the list will be hundreds of items long.

Reply:
Please give a concrete example @ljharb

We have ~ 60-80 across our orgs and none apply to a glob.

Likewise I have hundreds in my personal account and a glob would do nothing for me there either.

Reply:
github.com/tc39, for example, which has proposal repos starting with “proposal-“.
github.com/ossf, which has working groups staring with wg-, SIGs starting with sig-, etc.
github.com/inspect-js, which has common prefixes like has-, is-, and common suffixes like -equal

just to name a few.

Reply:
@ljharb thanks for the extra context.

If you'd like this, please can you ask for it in a new issue?

Reply:
I can, but as soon as this PR is released it becomes a breaking change, which is why it would have been best included at the outset.